### PR TITLE
Brennan Ayers

### PR DESCRIPTION
**Pull request by BrennanAyers**
Iteration 4 done.

### CODE REVIEW

File: lib/character.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Character
+  attr_reader :name, :actor, :salary
+
+  def initialize(hash)
+    @name = hash[:name]
+    @actor = hash[:actor]
+    @salary = hash[:salary]
+  end
+
```
Comment (ameseee) [inline lib/character.rb]: beep delete me

File: lib/network.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    @shows.each do |show|
+      show.characters.each do |character|
+        character_hash[character.actor] = character.salary
+      end
+    end
+    character_hash
+  end
+
+  def highest_paid_actor
+    payroll
```
Comment (ameseee) [inline lib/network.rb]: You don't need to call payroll here, it's doing nothing.

File: lib/show.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+  def initialize(name, creator, characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+    @total_salary = total_salary_equation
+  end
+
+  def total_salary_equation
+    show_salary = 0
```
Comment (ameseee) [inline lib/show.rb]: I'd recommend looking into using `.sum` to handle this!

**Comment by ameseee** [comment]
@BrennanAyers really nice job - a couple comments for your review that should help you clean up a few spots😊

File: lib/network.rb (hunk 2 above)
Comment (BrennanAyers) [reply lib/network.rb]: 😅

**Pull request closed without merging**